src/boards/controls/GPIOButtonControls.cpp

```diff
@@ -126,15 +126,16 @@ void GPIOButtonControls::setup_deep_sleep()
     rtc_gpio_set_direction(gpio_up, RTC_GPIO_MODE_INPUT_ONLY);
     rtc_gpio_pulldown_en(gpio_up);
 
-    rtc_gpio_init(gpio_down);
+    /* rtc_gpio_init(gpio_down);
```

Comment:
This should be made more wise in my own board Cinwrite: Extra GPIOS: 4 (From which only IO21 is an RTC GPIO)